[2025-07-09T22:09:34.501+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-09T22:09:34.531+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:09:32.688217+00:00 [queued]>
[2025-07-09T22:09:34.547+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:09:32.688217+00:00 [queued]>
[2025-07-09T22:09:34.548+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-09T22:09:34.570+0000] {taskinstance.py:2890} INFO - Executing <Task(PythonOperator): list_files_in_s3_bucket> on 2025-07-09 22:09:32.688217+00:00
[2025-07-09T22:09:34.577+0000] {standard_task_runner.py:72} INFO - Started process 230837 to run task
[2025-07-09T22:09:34.583+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'list_s3_files', 'list_files_in_s3_bucket', 'manual__2025-07-09T22:09:32.688217+00:00', '--job-id', '20', '--raw', '--subdir', 'DAGS_FOLDER/aws.py', '--cfg-path', '/tmp/tmpcsb6c__r']
[2025-07-09T22:09:34.587+0000] {standard_task_runner.py:105} INFO - Job 20: Subtask list_files_in_s3_bucket
[2025-07-09T22:09:34.654+0000] {task_command.py:467} INFO - Running <TaskInstance: list_s3_files.list_files_in_s3_bucket manual__2025-07-09T22:09:32.688217+00:00 [running]> on host dd8713d3b969
[2025-07-09T22:09:34.798+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='list_s3_files' AIRFLOW_CTX_TASK_ID='list_files_in_s3_bucket' AIRFLOW_CTX_EXECUTION_DATE='2025-07-09T22:09:32.688217+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-09T22:09:32.688217+00:00'
[2025-07-09T22:09:34.800+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-09T22:09:34.842+0000] {base.py:84} INFO - Retrieving connection 'aws_s3'
[2025-07-09T22:09:34.843+0000] {connection_wrapper.py:325} INFO - AWS Connection (conn_id='aws_s3', conn_type='aws') credentials retrieved from login and password.
[2025-07-09T22:09:36.715+0000] {logging_mixin.py:190} INFO - Files found in bucket:
[2025-07-09T22:09:36.717+0000] {logging_mixin.py:190} INFO - source1/
[2025-07-09T22:09:36.718+0000] {logging_mixin.py:190} INFO - source1/Aemf1.csv
[2025-07-09T22:09:36.725+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-09T22:09:36.741+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-09T22:09:36.741+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=list_s3_files, task_id=list_files_in_s3_bucket, run_id=manual__2025-07-09T22:09:32.688217+00:00, execution_date=20250709T220932, start_date=20250709T220934, end_date=20250709T220936
[2025-07-09T22:09:36.810+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-09T22:09:36.830+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-09T22:09:36.831+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
